src/components/app-content-renderer/virtual-assistant.js

```diff
@@ -0,0 +1,30 @@
+import { useFlag } from '@unleash/proxy-client-react';
+import React from 'react';
+import { ScalprumComponent } from '@scalprum/react-core';
+
+import './virtual-assistant.scss';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+
+function VirtualAssistant() {
+  const showVirtualAssistant = useFlag(
+    'platform.landing-page.virtual-assistant'
+  );
+  const chrome = useChrome();
+
+  // Disable it for prod, any stable environment or if the feature flag is off
+  if (chrome.isProd() || !chrome.isBeta || !showVirtualAssistant) {
+    return null;
+  }
+
+  return (
+    <div className="virtualAssistant astro__landing-page">
+      <ScalprumComponent
+        scope="virtualAssistant"
+        module="./AstroVirtualAssistant"
+        fallback={null}
```

Comment:
You should also give this an `ErrorComponent` of `null`. The reason being, if it crashes there is a default error component that can cover half of the screen.

You can test this by blocking some requests in your browser specific to the VA JS assets for example.

Reply:
Used `Fragment` instead of `null` . The later was giving an error about cloning `null`.

Tested adding this route . There wasn't any service running on that port

```js
  routes: {
    '/apps/virtual-assistant/': { host: 'http://localhost:5555' },
  },
```

Reply:
Right yeah, we are clonning the error node.

:+1: